### PR TITLE
feat: switch between organization.

### PR DESCRIPTION
**Pull request by deji2ghost**
# Description
[FE] Develop UI for Organization Switching
Description:

Develop the user interface to display a list of organizations a user belongs to, allowing them to select and switch between organizations.

Ensure that the UI reflects the selected organization's context accurately across all relevant sections of the application.
Capture 6

**Closes #issue_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

File: src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx [hunk 1]
```diff
@@ -14,21 +15,48 @@ import {
...
 import { useLocalStorage } from "~/hooks/use-local-storage";
+import { auth } from "~/lib/auth";
 import { CreateOrganization } from "../../create-organization";
 
+const getCurrentOrganization = async (orgId: string) => {
+  const payload = { isActive: true };
+  try {
+    const session = await auth();
+    const response = await axios.put(
+      `https://api-csharp.boilerplate.hng.tech/api/v1/users/organisations/${orgId}`,
```
Comment (phoenixdahdev) [inline src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx:26]: so in the case of other teams what will this do

Comment (phoenixdahdev) [inline src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx:26]: and who approved this endpoint

**Review by phoenixdahdev — CHANGES_REQUESTED** [review]

File: src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx (hunk 1 above)
Comment (deji2ghost) [reply src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx:26]: backend guy told me it was merged chief so i went on with the endpoint

Comment (deji2ghost) [reply src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx:26]: this keeps track of the active account switched to on the organization

**Pull request closed without merging**